### PR TITLE
Create a Rust project managed with cargo

### PR DESCRIPTION
**Pull request by mkirilly**
Added a single file to print out "Hello, GRN world!".

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,8 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+
+# Added by cargo
+
+/target
```
Comment (korommatyi) [inline .gitignore:15]: This is already there at line 3.

File: Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "grn_simulator"
+version = "0.1.0"
+authors = ["korommatyi", "mkirilly", "gabormate"]
```
Comment (korommatyi) [inline Cargo.toml:4]: Could you add `hannagabor` as well please?

**Review by korommatyi — APPROVED** [review]
Thanks!

**Comment by mkirilly** [comment]
I started a new pull request that incorporates the changes you requested

**Pull request closed without merging**